content/docs/12.5-palette-cli/20-commands.md

```diff
@@ -25,7 +25,14 @@ The complete list of subcommands is:
   * `help`       -   Help with any command.
 
 
-  * `pde`        -   Palette Developer Experience.
+  * [`login`](#login)      -  Login to Palette.
```

Comment:
@karl-cardenas-coding I wonder if we should change this to say 'Palette login' so that 'Login' isn't confused with the verb _to log in_.

Reply:
It should match the command as it appears in the CLI.